src/impact/capability-representation.md

```diff
@@ -0,0 +1,75 @@
+## Capability representation in memory
+
+Underlying implementations of CHERI are diverse, spanning 32-bit
+microcontrollers (such as Microsoft's CHERIoT) to 64-bit server-class
+processors (such as Arm's Morello).
+CHERI C/C++ provide broad flexibility for implementations to represent
+capability metadata in the ways most suitable to their individual
+requirements.
+One specific area in which CHERI implementations may differ is in the specific
+in-memory representations of capabilities, due to not just different address
+sizes, but also different tradeoffs around bounds compression, permissions,
+and so on.
+
+CHERI C/C++ in general expect that capabilities will be accessed via pointer
+types, with operations such as dereferencing a pointer or performing pointer
+arithmetic implemented by compiler-generated code.
+Broadly, a capability consists of three parts: An address, inline metadata,
+and a validity tag.
+When stored in memory, CHERI capabilities are twice the size of the native
+address type (e.g., 128 bits on 64-bit systems, and 64 bits on 32-bit
+systems), in addition to an unaddressable tag bit.
+Tag storage is capability aligned, and hence capabilities must themselves be
```

Comment:
Saying the tag storage is aligned feels odd. It's somewhat true with ECC-bit style implementations, but not really for a tag cache. Maybe something like "There is one tag per capability aligned region of memory, and ..."?

Reply:
I agree, that suggested wording sounds much clearer to me.

Reply:
I think I meant simply the abstraction of tag storage, but I can see that some readers might find this confusing if they are willing to think about the microarchitecture :-). I'll switch to @brooksdavis's proposed wording.